Content.Server/ParticleAccelerator/EntitySystems/ParticleAcceleratorSystem.ControlBox.cs

```diff
@@ -175,7 +175,8 @@ or ParticleAcceleratorPowerState.Level1
 
             _adminLogger.Add(LogType.Action, impact, $"{ToPrettyString(player):player} has set the strength of {ToPrettyString(uid)} to {strength}");
 
-
+            // Frontier: Admins do not need to be extra alerted for high PA levels and alarms startle them easily.
+            /*
```

Comment:
this may be better accomplished through changing the default cvar value. Would create fewer future conflicts, and still preserve the behavior.

https://github.com/new-frontiers-14/frontier-station-14/blob/e8fa9168046acfa2a09043b41b0a7809f4aa5051/Content.Shared/CCVar/CCVars.Admin.cs#L116C1-L117C110 

can change it from 5 to 6 and see if that will stop the alert for us.